src/diffusers/utils/peft_utils.py

```diff
@@ -68,3 +70,98 @@ def recurse_remove_peft_layers(model):
                 torch.cuda.empty_cache()
 
     return model
+
+
+def scale_lora_layers(model, lora_weightage):
```

Comment:
```suggestion
def scale_lora_layers(model, weight):
```

To me the name `lora_weightage` is a bit weird, could we just name it `scale` or `weight`